packages/manager/modules/pci/package.json

```diff
@@ -34,6 +34,7 @@
     "@ovh-ux/manager-config": "^2.0.0 || ^3.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
     "@ovh-ux/manager-error-page": "^0.0.0",
+    "@ovh-ux/manager-models": "^1.0.0",
```

Comment:
manager-models is agnostic so I think you can add it as a `dependency`

Reply:
Removed, not needed anymore.

Reply:
Dependency still declared, is it needed finally ?

Reply:
Yes, i need it.